libs/model-assessment/src/lib/ModelAssessmentDashboard/utils/addTabMessage.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+
+import { GlobalTabKeys } from "../ModelAssessmentEnums";
+
+export function addTabMessage(tab: GlobalTabKeys): string {

```

Comment:
Could you add a unit test for this function?

Reply:
![image](https://user-images.githubusercontent.com/76190371/192873309-7b2936d6-a657-4618-aa3e-e969be7966df.png)